configs/inspection/Square.xml

```diff
@@ -2,5 +2,6 @@
 <inspections version="1.0" is_locked="false">
   <option name="myName" value="Square" />
   <inspection_tool class="UnnecessarySemicolon" enabled="true" level="ERROR" enabled_by_default="true" />
+  <inspection_tool class="FallthruInSwitchStatement" enabled="true" level="ERROR" enabled_by_default="true" />
```

Comment:
It will

On Wed, Oct 7, 2015 at 3:06 PM Davis Gallinghouse <notifications@github.com>
wrote:

> In configs/inspection/Square.xml
> <https://github.com/square/java-code-styles/pull/18#discussion_r41432059>:
>
> > @@ -2,5 +2,6 @@
> >  <inspections version="1.0" is_locked="false">
> >    <option name="myName" value="Square" />
> >    <inspection_tool class="UnnecessarySemicolon" enabled="true" level="ERROR" enabled_by_default="true" />
> > +  <inspection_tool class="FallthruInSwitchStatement" enabled="true" level="ERROR" enabled_by_default="true" />
>
> Yeah, good point, I'll do that. Do you know if GitHub will correctly
> handle me amending and force-pushing this commit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/java-code-styles/pull/18/files#r41432059>.
>